src/ashley/editor/__init__.py

```diff
@@ -0,0 +1,31 @@
+"""
+This package contains utilities for Ashley's WYSIWYG editor.
+"""
+
+import json
+
+from draftjs_exporter.html import HTML
+
+from ..defaults import (
+    DRAFT_EXPORTER_BLOCK_MAP,
+    DRAFT_EXPORTER_COMPOSITE_DECORATORS,
+    DRAFT_EXPORTER_ENGINE,
+    DRAFT_EXPORTER_ENTITY_DECORATORS,
+    DRAFT_EXPORTER_STYLE_MAP,
+)
+
+DRAFTJS_EXPORTER = HTML(
+    {
+        "entity_decorators": DRAFT_EXPORTER_ENTITY_DECORATORS,
+        "composite_decorators": DRAFT_EXPORTER_COMPOSITE_DECORATORS,
+        "block_map": DRAFT_EXPORTER_BLOCK_MAP,
+        "style_map": DRAFT_EXPORTER_STYLE_MAP,
+        "engine": DRAFT_EXPORTER_ENGINE,
+    }
```

Comment:
Have you thought of having just one setting `DRAFTJS_EXPORTER` and pass the whole object since it is a dictionary? It can still be configured with an environment variable using json.

Reply:
Yep, I've thought about this. But I preferred this strategy (with 4 distinct settings) because most settings (like `block_map` and `sytle_map`) have adequate default values and it would be cumbersone to have to redefine the whole settings object just to change one value.

Another strategy could have been to merge the custom settings object with a default one, but i think it adds to much "magic" :mage_man: .

Reply:
@madmatah not sure it's magic from the user's point of view: if you redefine the setting it is overriden, if you don't there is a default...

Reply:
@sampaccoud Ok let's try, then (see commit 4f6a57d340430368301259f1d98265ba5cf317f0 )